### ISSUE TITLE
Zigbee lightstrip responds to Phoscon app but not Homebridge-hue (light bulbs are fine)

### ISSUE THREAD

**Issue by Jafalex** [ebaauw/homebridge-hue#636]
#### Issue
I have 3 Gledopto bulbs and 1 Gledopto lightstrip (all are Zigbee).
I have a new Conbee II so I've moved the above lights from my Philips Hue bridge to Conbee/DeCONZ/Phoscon. Things work as expected when using the Phoscon app.

When using Home however, things work somewhat OK. Bulbs are much slower than with the Philips Hue bridge, but it works. With the lightstrip though, I haven't been able to turn it on/off even once. The console show everything as expected, at least I think so.

Note: I'm running Homebridge in Docker on Raspbian (Rpi 4B).

#### Log Messages
Working Zigbee bulb
```
[2/27/2020, 7:54:36 PM] [Hue] ConbeeGW: gateway request 516: ok
[2/27/2020, 7:54:36 PM] [Hue] Spare room: state changed event: {"alert":null,"bri":254,"colormode":"xy","ct":156,"hue":46080,"on":false,"reachable":true,"sat":245,"xy":[0.2144,0.076]}
[2/27/2020, 7:54:37 PM] [Hue] ConbeeGW: gateway request 517: get /sensors
[2/27/2020, 7:54:37 PM] [Hue] ConbeeGW: gateway request 517: ok
[2/27/2020, 7:54:37 PM] [Hue] ConbeeGW: gateway request 518: get /lights
[2/27/2020, 7:54:37 PM] [Hue] ConbeeGW: gateway request 518: ok

```
Not working Zigbee lightstrtip
```
[2/27/2020, 7:56:00 PM] [Hue] Couch strip: homekit on changed from 0 to 1
[2/27/2020, 7:56:02 PM] [Hue] ConbeeGW: gateway request 551: get /sensors
[2/27/2020, 7:56:02 PM] [Hue] ConbeeGW: gateway request 551: ok
[2/27/2020, 7:56:02 PM] [Hue] ConbeeGW: gateway request 552: get /lights
[2/27/2020, 7:56:02 PM] [Hue] ConbeeGW: gateway request 552: ok

```
As you can see with the lightstrip I never see the "state changed event:"

Homekit then shows the lightship tile as "not responding" until I trigger it again, but the light itself never changes its state.

So it's hard to identify the issue since the log don't seem to show anything particularly helpful, II think?

#### Debug Files
Not sure how to do that when using Docker?

Thanks in advance for any help!

**Comment by ebaauw** [comment]
Please capture and attach a full debug log, see https://github.com/ebaauw/homebridge-hue#debug-log-file.

**Comment by Jafalex** [comment]
Hi Erik, thanks for your quick response! 
I've tried to get those debug files but not sure how to do so using Docker, is it possible?

**Comment by ebaauw** [comment]
Should be, but I don’t know the details.  I don’t use Docker.

**Comment by Jafalex** [comment]
Therefore 
```$ sudo journalctl -au homebridge```
returns
```
-- Logs begin at Mon 2020-02-24 23:32:24 NZDT, end at Thu 2020-02-27 20:07:51 NZDT. --
-- No entries --
```

**Comment by Jafalex** [comment]
Ok I'll try to export the logs manually, I'll be right back

**Comment by Jafalex** [comment]
Ok here we go, I've isolated the logs from your plugin only, to make it a bit easier to digest. Hope this contains what you need.
Cheers
[homebridge.log.gz](https://github.com/ebaauw/homebridge-hue/files/4260040/homebridge.log.gz)

**Comment by Jafalex** [comment]
Going through that json, the strip seems to be ```"reachable":true,``` and reports similar config to the working bulbs, I'm really confused as to what's going on here.

**Comment by Jafalex** [comment]
A couple more infos on the products...

![2811799F-2E29-4114-BB90-9196C46EEC79](https://user-images.githubusercontent.com/5572100/75428170-9f4b9880-59ac-11ea-8708-eb2be4b895b0.png)
![D690ED0F-D958-497B-9E2E-F92B7B371A19](https://user-images.githubusercontent.com/5572100/75428182-a4104c80-59ac-11ea-82c7-a7db874ca435.png)

**Comment by Jafalex** [comment]
I did add both log files above, is there anything else I can provide to help diagnose the issue?

**Comment by ebaauw** [comment]
homebridge-hue doesn't issue a `put` to the gateway, after you change the strip in HomeKit.  Very odd.  I still have a GL-C-008 lying around and can reproduce the issue.

**Comment by ebaauw** [comment]
Could you try v0.11.56-1, install thru `sudo npm -g i homebridge-hue@beta`?

**Comment by Sciphere** [comment]
I had the same issue with the Zigbee lightstrip controller. Just installed homebridge-hue v0.11.56-2  (beta) and it seems to have fixed the issue with my controller!

**Comment by Jafalex** [comment]
Thanks @Ebaauw for investigating and already having provided a fix! Genius!

Considering I’m running homebridge in Docker I’m not sure I can update to Hue beta, so I might have to wait until it’s officially deployed. 
I’ll update this thread if I find a way.

**Comment by ebaauw** [comment]
Just released v0.11.56.

**Comment by Jafalex** [comment]
Working like a charm straight away, and everything is also faster than before. Thank you so much for your amazing support!
